zetaclient/chains/bitcoin/observer/witness.go

```diff
@@ -173,14 +177,12 @@ func tryExtractInscription(tx btcjson.TxRawResult, logger zerolog.Logger) []byte
 	return nil
 }
 
-func isValidAmount(
-	incoming float64,
-	minimal float64,
-) (bool, float64) {
-	if incoming < minimal {
-		return false, 0
+// DeductDepositorFee returns the inbound amount after deducting the depositor fee.
```

Comment:
```suggestion
// DeductDepositorFee returns the inbound amount after deducting the depositor fee.
// returns an error if the deposited amount is lower than the depositor fee.
```

Reply:
updated in same commit https://github.com/zeta-chain/node/pull/3358/commits/3b44dc6cf18412c681ca31b4917fdcb8fccf720f